### PR TITLE
PRIME-2370 Update Enrolment UX

### PR DESCRIPTION
**Pull request by bergomi02**
https://pr-2279.pharmanetenrolment.gov.bc.ca/info

### CODE REVIEW

**Comment by neophyte57** [comment]
From "Enrolment Review" screen for enrollee with no College Licence 
1. Click to edit "Job Site Information"
2. Click Back
3. **Expected screen is "Enrolment Review" but instead it is "College Licence Information"**

**Comment by bergomi02** [comment]
> From "Enrolment Review" screen for enrollee with no College Licence
> 
> 1. Click to edit "Job Site Information"
> 2. Click Back
> 3. **Expected screen is "Enrolment Review" but instead it is "College Licence Information"**

this should be fixed.

File: prime-angular-frontend/src/app/modules/enrolment/enrolment.routes.ts [hunk 1]
```diff
@@ -40,6 +42,16 @@ export class EnrolmentRoutes {
   public static ENROLMENT = 'enrolment';
   public static ABSENCE_MANAGEMENT = 'absence-management';
 
+  // Step display text
+  public static SETP_CONTACT = 'Contact';
+  public static STEP_CARE_SETTING = 'Care Setting';
+  public static STEP_COLLEGE = 'College /<br/>Job Info';
+  public static STEP_SELF_DECLARATION = 'Self-Declaration';
+  public static STEP_REVIEW = 'Review';
+  public static STEP_TOA = 'Term of Access';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/enrolment/enrolment.routes.ts:51]: Should be "Terms of Access"

image

File: prime-angular-frontend/src/app/modules/enrolment/enrolment.routes.ts [hunk 2]
```diff
@@ -40,6 +42,16 @@ export class EnrolmentRoutes {
   public static ENROLMENT = 'enrolment';
   public static ABSENCE_MANAGEMENT = 'absence-management';
 
+  // Step display text
+  public static SETP_CONTACT = 'Contact';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/enrolment/enrolment.routes.ts:46]: Typo in `SETP_CONTACT`

File: prime-angular-frontend/src/app/modules/enrolment/pages/bcsc-demographic/bcsc-demographic.component.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ import { UtilsService } from '@core/services/utils.service';
 import { FormUtilsService } from '@core/services/form-utils.service';
 import { Enrolment } from '@shared/models/enrolment.model';
 import { ToggleContentChange } from '@shared/components/toggle-content/toggle-content.component';
+import { IStep } from '@shared/components/progress-indicator/progress-indicator.component';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/enrolment/pages/bcsc-demographic/bcsc-demographic.component.ts:16]: Is this needed?

**Review by neophyte57 — APPROVED** [review]
GTG  👍

**Pull request merged**